docs/overview.md

```diff
@@ -1,6 +1,86 @@
-# Overview
+page_title: Docker Registry Service 2.0
+page_description: Introduces the docker registry service
+page_keywords: registry, service, images, repository
 
-**TODO(stevvooe):** Table of contents.
+# Docker Registry Service 2.0
 
-**TODO(stevvooe):** Include a full overview of each component and dispatch the
-user to the correct documentation.
+The Docker Registry Service stores and distributes Docker images. The majority
+of Docker users pull images from Docker's own public registry instance.
```

Comment:
I'm still not sure the "hub" is a good way to introduce the registry. The hub does a lot of things the registry doesn't do and comparing them may give users the idea that this code base will do what the hub does for them. We can use the hub as an example of something that has a registry.